alembic/README

```diff
@@ -0,0 +1 @@
+Generic single-database configuration.
```

Comment:
You can remove it or include instructions on how to set up project and which commands to use. 
Or maybe even short description of the project.